lib/linkerd2-dns-name/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "linkerd2-dns-name"
```

Comment:
TIOLI: it looks like the `-addr` and `-dns-name` crates are small enough, unlikely to change, and related enough, that perhaps they ought to be combined?